config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```diff
@@ -187,3 +187,51 @@ periodics:
           limits:
             cpu: "2"
             memory: "6Gi"
+
+  # This jobs runs e2e.test with a focus on long running tests for all alpha/beta storage features on a kind cluster
+  - name: pull-kubernetes-e2e-storage-kind-alpha-beta-features-slow
+    cluster: eks-prow-build-cluster
+    optional: true
+    always_run: false # manual for testing new configurations
```

Comment:
this is invalid, it will be triggered on the `interval` below

https://docs.prow.k8s.io/docs/jobs/

see also config/jobs/README.md https://github.com/kubernetes/test-infra/tree/master/config/jobs

Reply:
I followed the session for presubmits, but it asks me to add cron, internal or minimum_interval, is this correct?